### PR TITLE
Added aliases

### PR DESCRIPTION
**Pull request by pecey**
Added aliases so as to enable queries like `sublime cheatsheet` and `sublime text 2 cheatsheet` to work

IA : https://duck.co/ia/view/sublime_text_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pecey Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/sublime-text.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
         "sourceName": "Cheatography",
         "sourceUrl": "http://www.cheatography.com"
     },
+    "aliases": [
+         "sublime",
+         "sublime text",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/sublime-text.json]: We can remove this alias - it it covered by the file name.

Comment (pecey) [inline share/goodie/cheat_sheets/json/sublime-text.json]: Okay sure.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**